src/components/User/UserAvatar.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+
+import { PreviewCamera, PreviewEmote } from '@dcl/schemas'
+import { WearablePreview } from 'decentraland-ui'
+
+import useProfile from '../../hooks/useProfile'
+
+import './UserAvatar.css'
+
+interface Props {
+  address?: string
+}
+
+export default function UserAvatar({ address }: Props) {
+  const [wearablePreviewController, setWearablePreviewController] = useState<any>()
```

Comment:
there is no defined type?

Reply:
The type is `IPreviewController` but I couldn't import it 😞